flutter/lib/src/sentry_flutter_options.dart

```diff
@@ -158,46 +161,18 @@ class SentryFlutterOptions extends SentryOptions {
   /// available in the Flutter environment. This way you get more detailed
   /// information where available.
   void enableBreadcrumbTrackingForCurrentPlatform() {
-    // defaultTargetPlatform returns the platform this code currently runs on.
-    // See https://api.flutter.dev/flutter/foundation/defaultTargetPlatform.html
-    //
-    // To test this method one can set
-    // foundation.debugDefaultTargetPlatformOverride as one likes.
-    configureBreadcrumbTrackingForPlatform(foundation.defaultTargetPlatform);
+    configureBreadcrumbTrackingForPlatform(platformChecker);
   }
 
   /// You should probably use [enableBreadcrumbTrackingForCurrentPlatform].
   /// This should only be used if you really want to override the default
   /// platform behavior.
   @foundation.visibleForTesting
-  void configureBreadcrumbTrackingForPlatform(
-      foundation.TargetPlatform platform) {
-    // Bacause platform reports the Operating System and not if it is running
-    // in a browser. So we have to check if this is Flutter for web.
-    // See https://github.com/flutter/flutter/blob/c5a69b9b8ad186e9fce017fd4bfb8ce63f9f4d13/packages/flutter/lib/src/foundation/_platform_web.dart
-    if (foundation.kIsWeb) {
-      // Flutter for web has no native integration, just use the Flutter
-      // integration.
+  void configureBreadcrumbTrackingForPlatform(PlatformChecker checker) {
```

Comment:
We could no remove this method as it is not needed anymore for testing. The relevant code can then be moved to  `enableBreadcrumbTrackingForCurrentPlatform()`.

Reply:
That makes sense to me